packages/lit-dev-content/src/global/hydrate-common-components.ts

```diff
@@ -63,6 +49,72 @@ const activateMobileDetails = async () => {
   );
 };
 
+// hydrate the search modal
+
+// There are 2 modal buttons on the page. In the header and in the mobile nav.
+// We need to open the one in the current view or else it'll open a hidden
+// modal that makes the dom inert.
+const searchModals = document.querySelectorAll(
+  'litdev-search-modal'
+) as NodeListOf<LitDevSearchModal>;
+let modalHydrated = false;
+
+const hydrateSearchModal = async () => {
+  if (modalHydrated) return;
```

Comment:
Does `modalHydrated` need to be set to true below this line?

Reply:
Excellent catch!